examples/using-MDX/src/components/seo.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * SEO component that queries for data with
```

Comment:
This component isn't really relevant to an MDX example, so I'd recommend removing it to focus on the important parts.